docs/release-promotion/desktop/howto-rc.md

```diff
@@ -37,22 +47,26 @@
       },
       "fields": {
         "actions": "showURL",
-        "openURL": "https://www.mozilla.org/%LOCALE%/firefox/59.0/whatsnew/?oldversion=%OLD_VERSION"
+        "openURL": "https://www.mozilla.org/%LOCALE%/firefox/59.0/whatsnew/?oldversion=%OLD_VERSION%"
       }
     }
   ]
 ```
 
-The block above says that all responses constructed with this blob should include detailsURL and type (because the first "for" block is empty), while only requests matching locales and versions from the second "for" block should get "actions" and "openURL" in their response. The list of locales and WNP URL should be whatever you received from Product before you began this process.
+The block above says that all responses constructed with this blob should include `detailsURL` and `type` (because the first `for` block is empty), while only requests matching `locales` and `versions` from the second `for` block should get `actions` and `openURL` in their response.
+The list of locales and WNP URL should be whatever you received from Product before you began this process.
+
+**A word on the** `locales` The final number of locales that are to be present within a certain WNP depends on each release. There's a certain cut-off date for each release, by which localized contents can be submitted. Once passed the cut-off date
```

Comment:
I'm skeptical that this will remain true for very long, but let's see what happens.